ui/components/multichain/asset-picker-amount/asset-picker-modal/AssetList.tsx

```diff
@@ -61,19 +62,14 @@ export default function AssetList({
       hideLabel: true,
     });
 
-  const swapsBlockedTokens = useSelector(getSwapsBlockedTokens);
-  const memoizedSwapsBlockedTokens = useMemo(() => {
-    return new Set(swapsBlockedTokens);
-  }, [swapsBlockedTokens]);
```

Comment:
minor: any reasons why we don't store swapsBlockedTokens as a set?

Reply:
We can circle back to this in a follow-up PR after a discussion to keep refactors at a minimum here